image_builder/data/toolchain.cmake

```
@@ -0,0 +1,16 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
+
+# TODO: this hard code is not great, but solving it comprehensively is kind of
+# difficult (maybe dockcross? but that has its own drawbacks..)
+set(CMAKE_SYSROOT /tmp/rpi4-image-build)
```

We can use here and ENV variable something like:

```suggestion
set(CMAKE_SYSROOT $ENV{SYSROOT})
```

---

I would rather just leave it hard coded for now. That whole path is not defined particularly well for now. I would like to get this to work before fixing that path across everything.

---

For sure!